StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/NetworkingOTPView/NetworkingOTPView.swift

```diff
@@ -30,15 +30,26 @@ protocol NetworkingOTPViewDelegate: AnyObject {
 
 final class NetworkingOTPView: UIView {
 
+    enum TestModeValues {
+        static let otp = "000000"
```

Comment:
subjective: because this is only used in one place, you could inline it too, but I break this rule sometimes too

the thought process is that its simpler to inline (ex. if both this solution, and inline solution, can be equally considered valid, then one could say that "in this battle of subjectivity" [the simplest solution is the winner](https://en.wikipedia.org/wiki/Occam%27s_razor) haha)

Reply:
I'm fine with either approach, but I will leave this as is for now since this is the same pattern that was used for the Manual Entry flow